scripts/build-translations.js

```diff
@@ -26,4 +26,4 @@ const translationsMap = supportedLanguages
 
 const translationsContent = `window.OONITranslations = ${JSON.stringify(translationsMap)}`
 writeFileSync(`${LANG_DIR}/translations.js`, translationsContent)
-console.log(`> Wrote translations to: ${LANG_DIR}/translations.js`)
```

Comment:
I think we should be keeping these `console.log` lines inside of the scripts, since it's expected for them to print some output to the user of them.